cypress/integration/userCanSortProductsByCategories.feature.js

```diff
@@ -0,0 +1,27 @@
+describe("User can see list of", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3001");
+  });
+
+  describe("products", () => {
+    before(() => {
+      cy.server();
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/products",
+        response: "fixture:products.json",
+      });
+
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/categories/**",
+        response: "fixture:veggie_request.json",
+      });
+    });
+
+    it("sorted by categories", () => {
+      cy.get('[data-cy="dropdown"]').click();
+      cy.get("[role='option']").contains('Veggie').click();
```

Comment:
I need an assertion here, just clicking isn't enough

Reply:
Noted!

Reply:
We have added this now!